datalake/src/main/java/com/sequenceiq/datalake/flow/detach/event/DatalakeResizeFlowChainStartEvent.java

```diff
@@ -36,4 +38,12 @@ public String getBackupLocation() {
     public String selector() {
         return SDX_RESIZE_FLOW_CHAIN_START_EVENT;
     }
+
+    @Override
+    public boolean equalsEvent(SdxEvent other) {
+        return isClassAndEqualsEvent(DatalakeResizeFlowChainStartEvent.class, other,
+                event -> Objects.equals(sdxCluster, event.sdxCluster)
```

Comment:
Wouldn't it be better to only check if the CRN is the same?

Reply:
Nevermind, it's the new sdxCluster that is replacingTheOther? It has a misleading name in the model then.

Reply:
Yes, its the new sdxCluster.

Reply:
The equals of SdxCluster uses the created timestamp which always make equals return false. I will fix this.